app/pages/Onboarding/ImportSeedFlow/index.js

```diff
@@ -14,6 +14,7 @@ import OptInAnalytics from '../OptInAnalytics';
 import { clientStub as aClientStub } from '../../../background/analytics/client';
 import { showError } from '../../../ducks/notifications';
 import SetName from '../SetName';
+import UpdateAccountDepth from "../UpdateAccountDepth";
```

Comment:
at component level the concept of depth (sing.) seems appropriate.  Why not reflect that in the UI?

just thinking from a hypothetical user's perspective, "I am going to change my Depth**s**" sounds awkward in my head